packages/react-conformance/src/defaultTests.tsx

```diff
@@ -82,6 +82,62 @@ export const defaultTests: TestObject = {
     });
   },
 
+  /** Component handles ref */
+  'component-handles-ref': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`handles ref`, () => {
+      try {
+        const { customMount = mount, Component, requiredProps, elementRefName = 'ref' } = testInfo;
+        const rootRef = React.createRef<HTMLDivElement>();
+        const mergedProps: Partial<{}> = {
+          ...requiredProps,
+          [elementRefName]: rootRef,
+        };
+
+        customMount(<Component {...mergedProps} />);
+
+        expect(rootRef.current).toBeDefined;
+        // Ref should resolve to an HTML element.
+        expect(rootRef.current?.getAttribute).toBeDefined;
```

Comment:
```suggestion
        expect(rootRef.current).toBeDefined();
        // Ref should resolve to an HTML element.
        expect(rootRef.current?.getAttribute).toBeDefined();
```

Reply:
I was wondering why this was passing for some tests; a lot more tests will need to disable `component-handles-ref`